deepchem/transformers/tests/test_transformers.py

```diff
@@ -64,6 +63,68 @@ def test_X_log_transformer(self):
 
     # Check that untransform does the right thing.
     np.testing.assert_allclose(log_transformer.untransform(X_t), X)
+ 
+  def test_y_log_transformer_select(self):
+    """Tests logarithmic data transformer with selection."""
+    multitask_dataset = self.load_feat_multitask_data()
+    dfe = pd.read_csv("../../models/tests/feat_multitask_example.csv")
```

Comment:
I think the issue is that you need to say
```
dfe = pd.read_csv(os.path.join(self.current_dir, "../../models/tests/feat_multitask_example.csv"))
```
There's no guarantee about where the test is executed on travis.